src/app/components/ui/ukhsa/Context/TopicBodyContext.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+
+import * as React from 'react'
+import { useState } from 'react'
+
+export function useTopicBodyFilters(initialFilters: string[] = ['Leicester', 'London', '6-in-1']) {
```

Comment:
Assuming this default array should be empty
```suggestion
export function useTopicBodyFilters(initialFilters: string[] = []) {
```